import-export-cli/impl/logger.go

```diff
@@ -23,6 +23,7 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"strings"
```

Comment:
Shouldn't this import come below "os" import?

Reply:
Fixed via https://github.com/wso2/product-apim-tooling/pull/966/commits/5584c0cda4fcd40ca92ceb0a8368f41672eba295